practice-two/src/styles/layouts/comment.scss

```diff
@@ -1,5 +1,5 @@
 .comment {
-  font-family: Satoshi;
+  font-family: $font-secondary;
   margin-bottom: toRem(49.81px);
   width: math.div(358, 390) *100%;
```

Comment:
Check code style. Why did you set the width like this `math.div(358, 390) *100%;`